front/components/items/Appointment/helper.ts

```diff
@@ -41,3 +46,21 @@ export const findMatchingAppointment = ({
             return item;
         }
     });
+
+export const setFirstAvaibleAppointmentDate = ({
+    chosenTherapist,
+    dataAppointmentWithTherapist,
+}: propsOfSetFirstAvaibleAppointmentDate) => {
+    if (dataAppointmentWithTherapist && chosenTherapist) {
+        const filteringByTherapist = dataAppointmentWithTherapist?.filter(
+            (item) => item.attributes.therapist.data.id === chosenTherapist
+        );
+        if (filteringByTherapist.length !== 0) {
+            const datesToNumber = filteringByTherapist?.map((item) =>
```

Comment:
Id find another name for the variable cos it doesn't say on what dates are you operating. Other way is to skip this variable and calculate this in return statement
```suggestion
            return new Date(Math.min(...filteringByTherapist?.map((item) =>
```